spec/mailers/abuse_mailer_spec.rb

```diff
@@ -87,11 +87,44 @@
    it "body text contains the comment" do
      expect(mail).to have_body_text(/#{report.comment}/)
    end
-
+   
    it "has the url of the page with abuse" do
      expect(mail.html_part).to have_body_text(report.url)
    end
   end
 
+  
+  context "abuse_reports contains creator and title if reporting a work" do
+   let(:work) {create(:work)}
+   let(:report) {create(:abuse_report, url: work_url(work.url)}
+   let(:mail) {AdminMailer.abuse_report(report.id)}
+
+   it "has the correct subject" do
+     expect(mail).to have_subject "[#{ArchiveConfig.APP_SHORT_NAME}] Admin Abuse Report"
+   end
+
+   it "delivers to the correct address" do
+     expect(mail).to deliver_to ArchiveConfig.ABUSE_ADDRESS
+   end
+   it "delivers from the correct address" do
+     expect(mail).to deliver_from("Archive of Our Own <#{ArchiveConfig.RETURN_ADDRESS}>")
+   end
+
+   it "body text contains the title of the reported work" do
+     expect(mail).to have_body_text(/#{report.metadata["title"]}/)
+   end
+
+   it "body text contains the author of the reported work" do
+     expect(mail).to have_body_text(/#{report.metadata["author"]}/)
+   end
+
+   it "body text contains the comment" do
+     expect(mail).to have_body_text(/#{report.comment}/)
+   end
+   
+   it "has the url of the page with abuse" do
+     expect(mail.html_part).to have_body_text(report.url)
+   end
+  end
```

Comment:
It's hard to tell in the diff, but this means you probably have too many `end`s or your `do...end` blocks aren't properly matched.